README.md

```diff
@@ -1,131 +1,29 @@
 Chitter Challenge
 =================
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+* A simple imitation of twitter that allows a user to sign in and tweet from their account. Allowssomeone not signed in to view the tweets but not write them. Other functionality includes signing up and logging out
+* This project was made using the Sinatra framework with ~Ruby. The database sytem was created with PostgreSQL and TablePlus.
 
-Challenge:
--------
+# Requirements
+ruby '3.0.0'
 
-As usual please start by forking this repo.
+# How to install
+- open terminal and execute `git clone https://github.com/eddiearnoldz/chitter-challenge.git`
+- bundle install the gem file by running `bundle install`
+- in terminal run `rackup`and visit  'http://localhost:9292/peeps'
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+# Installing the database
 
-Features:
--------
+- Connect to psql
+- Create the database using the psql command `CREATE DATABASE chitter_manager;`
+- Connect to the database using the pqsl command `\c chitter_manager;`
+- Run the query we have saved in the file 01_create_chitter_table_peeps.sql
+- Run the query we have saved in the file 02_create_chitter_table_users.sql
```

Comment:
Great README with instructions on how to connect to your database - really helpful to enable me to get your project up and running.